app/views/countries/partials/_protection-dials.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container section-padding--bottom">
```

Comment:
Views file format for rails should conventionally be `.html.erb` instead of just `.erb`